tests/integration/conftest.py

```diff
@@ -13,17 +13,18 @@ def capabilities(capabilities):
 @pytest.fixture
 def firefox_options(firefox_options):
     """Set Firefox Options."""
-    firefox_options.headless = True
+    # firefox_options.headless = True
     return firefox_options
 
 
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Oh interesting, I'm curious about this.  What does this do?

Reply:
This is so this fixture only runs once per test session. And ```autouse``` means it is used automatically. So this fixture runs automatically once per test session.

Reply:
Unless you are talking about the quotes, then that is just a python black package change.